### PR TITLE
S5 delete products

### PR DESCRIPTION
**Pull request by mclack**

### CODE REVIEW

File: __test__/ProductValidationService.test.js [hunk 1]
```diff
@@ -52,4 +52,25 @@ describe('validateStockQuantity', () => {
...
+    })
+    test('testFailureInvalidSku', () => {
+        expect(PVS.validateSku('BUN-14HL64MF95NE40NGL88')).toBe(false)
+    })
+    test('testFailureInvalidSku', () => {
+        expect(PVS.validateSku('<h1>14HL64MF95NE40</h1>')).toBe(false)
+    })
+    test('testFailureInvalidSku', () => {
+        expect(PVS.validateSku([])).toBe(false)
+    })
```
Comment (HannaSBarratt) [inline __test__/ProductValidationService.test.js:75]: Add test for empty string

File: assets/db/blackmarket.sql [hunk 2]
```diff
@@ -31,21 +31,22 @@ CREATE TABLE `products` (
...
+  `deleted` tinyint(1) DEFAULT '0',
   PRIMARY KEY (`id`),
   UNIQUE KEY `sku` (`sku`)
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 
 LOCK TABLES `products` WRITE;
 /*!40000 ALTER TABLE `products` DISABLE KEYS */;
 
-INSERT INTO `products` (`id`, `product-name`, `price`, `stock-quantity`, `sku`)
+INSERT INTO `products` (`id`, `product-name`, `price`, `stock-quantity`, `sku`, `deleted`)
```
Comment (HannaSBarratt) [inline assets/db/blackmarket.sql:42]: Don't allow null for deleted field
